master/docs/examples/git_gerrit.cfg

```diff
@@ -184,7 +184,15 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
                 })
 
 c['buildbotURL'] = 'http://buildbot.example.com/'
+
+# The 'GerritStatusPush' target is used to publish build results in Gerrit
+# To create message (with Code-Review) used function 'gerritReviewCB'
+# Notification about this message will be sent to to all (change owners,
+# reviewers, watchers and any user who has starred the change)
+# https://gerrit-documentation.storage.googleapis.com/Documentation/2.9/cmd-review.html#_options
+
 c['status'].append(status.GerritStatusPush(gerrit_url, gerrit_user,
+                                           notify='ALL',
```

Comment:
[nit] `ALL` is the default behaviour, right?  So I'd add a comment that "the code below would result in the same notifications as if `notify` parameter is not specified".